Project.toml

```diff
@@ -44,7 +44,7 @@ ClimaComms = "0.6.4"
 ClimaCore = "0.14.19"
 ClimaDiagnostics = "0.2.4"
 ClimaParams = "0.10.17"
-ClimaTimeSteppers = "0.7.39"
+ClimaTimeSteppers = "=0.7.38"
```

Comment:
```suggestion
ClimaTimeSteppers = "<0.7.38"
```

Reply:
If 0.7.39 is known to have a critical bug, the proper thing to do is release 0.7.40 with the changes reverted

Reply:
v0.8.0 has already been released - can we still go back and release 0.7.40?

Reply:
Yes, we can create a branch in climatimesteppers and release in that branch.

If we don't release it and know that 0.7.39 has a critical bug, every package (or prior versions of ClimaAtmos) with compat 0.7 will always get a buggy version. If the bug is not super serious, we can skip this

Reply:
@dennisYatunin could you revert the change and release CTS v0.7.40?

Reply:
The bug is definitely serious, but also is there an issue with just proceeding using CTS 0.8.0? I'm about to release 0.8.1 but 0.8.0 is ready to merge in

Reply:
We need ClimaAtmos on 0.8.1 by the end of today so I'm not sure if it's useful to make a new 0.7.40